app/views/layouts/_nav.html.erb

```diff
@@ -18,13 +17,13 @@
         <%= link_to t(".augury"), current_user_app("augury"), class: "nav-link" %>
       <% end %>
     </nav>
-      
+
     <nav class="navbar-nav dropdown">
       <h2 class="visually-hidden">Help Menu</h2>
       <a class="nav-link nav-link-icon dropdown-toggle" href="#" id="nav-help-dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
         <span class="material-icons align-middle">help</span>
       </a>
-      <div class="dropdown-menu dropdown-menu-end dropdown-menu-dark position-absolute bg-primary-dark shadow" aria-labelledby="nav-help-dropdown">
+      <div class="dropdown-menu dropdown-menu-end dropdown-menu-dark position-absolute bg-primary-dark shadow m-0" aria-labelledby="nav-help-dropdown">
```

Comment:
There was a slight 1px margin added to the dropdown. I prefer dropdowns to be margin-free.